pkg/agent/billing/billing.go

```diff
@@ -276,10 +275,10 @@ func (s *metricsState) drainEnqueue(
 		history.finalizeCurrentTimeSlice()
 
 		countInBatch += 1
-		enqueue(logAddedEvent(logger, billing.Enrich(now, hostname, countInBatch, batchSize, &billing.IncrementalEvent{
+		enqueue(logAddedEvent(logger, enrich(now, hostname, countInBatch, batchSize, &IncrementalEvent{
 			MetricName:     conf.CPUMetricName,
-			Type:           "", // set by billing.Enrich
-			IdempotencyKey: "", // set by billing.Enrich
+			Type:           "", // set by reporting.Enrich
+			IdempotencyKey: "", // set by reporting.Enrich
```

Comment:
nit: remove reference to `reporting`? (+ same below)

Reply:
dropped